alpine/Dockerfile

```diff
@@ -19,9 +19,13 @@ ENV BUILDKITE_AGENT_VERSION=${BUILDKITE_AGENT_VERSION} \
     PATH=$PATH:/buildkite/bin
 
 COPY ./scripts/docker/* /tmp/
+
 RUN ARCH=386 /tmp/install_buildkite.sh ${BUILDKITE_AGENT_VERSION} \
     && cp /tmp/entrypoint.sh /usr/local/bin/entrypoint.sh \
-    && rm -rf /tmp/*
+    && rm -rf /tmp/* \
+    && adduser -D -g '' buildkite \
+    && chown -R buildkite /buildkite
```

Comment:
When I run `./scripts/build.sh` locally the alpine edge container fails on this line. If I change it to something like `&& (chown -R buildkite /buildkite || true)` the build passes, but I'm pretty sure this isn't what we want.

I assume the `/tmp/install_buildkite.sh` script should create this directory but for some reason it's not with edge alpine. Is this a correct assumption?